manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml

```diff
@@ -55,6 +55,8 @@ spec:
             - "--nodeid=$(NODE_ID)"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--cloud-config=$(CLOUD_CONFIG)"
+            - "--mode=ephemeral"
```

Comment:
dont we need to support both modes, can we make it configurable ?

Reply:
it is configurable via flag --mode,  i need to update this to set mode=persistent, if flag is not specified default mode is persistent, as of k8s v1.15 driver can run only in one mode. when we update to 1.16 driver can run in both modes. we need to specify mode of the driver in CSIDriver object
https://github.com/kubernetes/kubernetes/pull/80568